src/components/HeaderComponent/index.js

```diff
@@ -0,0 +1,264 @@
+  const [loading, setLoading] = useState(false);
+  const [visibleObtener, setVisibleObtener] = useState(false);
+  const [visibleAnuncio, setVisibleAnuncio] = useState(false);
+  const [visibleStickers, setVisibleStickers] = useState(false);
+  const [ad, setAd] = useState(null);
+  const [obtainedStickers, setObtainedStickers] = useState([]);
+
+  const { token } = useSelector(state => state.auth);
+
+  const onClaimClick = async () => {
+    setLoading(true);
+    try {
+      setAd(await watchAd());
+    } catch (error) {
+      alert(error.message);
+    } finally {
+      setLoading(false);
+      setVisibleAnuncio(true);
+    }
+  };
+
+  const onCloseAd = async () => {
+    setLoading(true);
+    try {
+      setObtainedStickers(await obtainStickers(token));
+    } catch (error) {
+      alert(error.message);
+    } finally {
+      setLoading(false);
+      setVisibleStickers(true);
+    }
+  };
+
+  const onAdClick = () => {
+    const redirectUrl = getAdRedirectUrl(ad?.id);
+    console.log('LE HAZ DADO CLICK A UN ANUNCIO :D ', redirectUrl);
+    Linking.openURL(redirectUrl);
+  };
+
+  return (
+    <SafeAreaView style={styles.header}>
+      <Spinner visible={loading} textContent={'Cargando...'} />
+      {/* Ventana Emergente de Obtener Cromos */}
+      <ModalPopup visible={visibleObtener}>
+        <View style={{ alignItems: 'center' }}>
+          <View style={styles.modalHeader}>
+            <TouchableOpacity onPress={() => setVisibleObtener(false)}>
+              <Image source={botonX} style={{ height: 30, width: 30 }} />
+            </TouchableOpacity>
+          </View>
+        </View>
+        <View style={{ alignItems: 'center' }}>
+          <Image
+            source={sobre}
+            style={{
+              width: 250,
+              height: 250,
+              resizeMode: 'contain',
+            }}
+          />
+        </View>
+        <TouchableOpacity
+          style={styles.logInButton}
+          onPress={() => {
+            onClaimClick();
+          }}
+        >
+          <Text style={{ color: 'white', fontWeight: 'bold' }}>Reclamar</Text>
+        </TouchableOpacity>
+      </ModalPopup>
+
+      {/* Ventana Emergente de drop de Stickers */}
+      <ModalPopup visible={visibleStickers}>
+        <View style={{ alignItems: 'center' }}>
+          <View style={styles.modalHeader}>
+            <TouchableOpacity onPress={() => setVisibleStickers(false)}>
+              <Image source={botonX} style={{ height: 30, width: 30 }} />
+            </TouchableOpacity>
+          </View>
+        </View>
+        <View
+          style={{
+            justifyContent: 'center',
+            flexDirection: 'row',
+            flexWrap: 'wrap',
+          }}
+        >
+
+          {obtainedStickers ? (
+            obtainedStickers.map((sticker, i) => (
+              <View style={{marginVertical: 78}}>
+                <StickerTemplate sticker={sticker} key={i} />
+              </View>
+            ))
```

Comment:
Por que si cada elemento tiene un "key" me sale esto :S ?
![image](https://user-images.githubusercontent.com/22270579/206922453-1a3f99e2-6251-4060-a3b5-a3ee5f34b51e.png)

Reply:
aaaah, es por el view
```suggestion
            obtainedStickers.map((sticker, i) => (
              <View style={{marginVertical: 78}} key={'viewTemplate-'+i} >
                <StickerTemplate sticker={sticker} key={'stickerTemplate-'+i} />
              </View>
            ))
```
O quitale el key a sticker Template, pero no estoy seguro de que no lo necesite